### PR TITLE
fix(controls): Remove controls wrapper

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

File: src/lib/Controls.js [hunk 1]
```diff
@@ -35,13 +35,9 @@ class Controls {
     constructor(container) {
         this.containerEl = container;
 
-        const controlsWrapperEl = document.createElement('div');
-        controlsWrapperEl.className = 'bp-controls-wrapper';
-        controlsWrapperEl.setAttribute('data-testid', 'controls-wrapper');
```
Comment (jstoffan) [inline src/lib/Controls.js]: Make sure this isn't used by any end-to-end tests in other repos (elements, webapp, etc.)

File: src/lib/Controls.scss [hunk 2]
```diff
@@ -1,15 +1,6 @@
-.bp-controls-wrapper {
+.bp-controls {
     position: absolute;
     bottom: 25px;
-    left: 50%;
```
Comment (jstoffan) [inline src/lib/Controls.scss]: It doesn't seem like the control bar will be centered on the screen. Do we need to do something like `left: 50%; transform: translateX(-50%);`?

File: src/lib/_common.scss [hunk 3]
```diff
@@ -160,6 +160,7 @@ $header-height: 48px;
         display: flex;
         flex: 1 1 auto;
         align-items: center;
+        justify-content: center;
```
Comment (jstoffan) [inline src/lib/_common.scss]: What is the reason for this change? It seems pretty risky.

Comment (mxiao6) [reply src/lib/_common.scss]: > What is the reason for this change? It seems pretty risky.

so we can avoid using `left: 50%; transform: translateX(-50%);`

File: src/lib/Controls.scss (hunk 2 above)
Comment (mxiao6) [reply src/lib/Controls.scss]: > It doesn't seem like the control bar will be centered on the screen. Do we need to do something like `left: 50%; transform: translateX(-50%);`?

I did it in the parent container, which you think it's risky below

File: src/lib/Controls.js (hunk 1 above)
Comment (mxiao6) [reply src/lib/Controls.js]: > Make sure this isn't used by any end-to-end tests in other repos (elements, webapp, etc.)

Checked (elements, webapp, annotations) and there's no usage of it.
